packages/app-desktop/gui/NoteListControls/NoteListControls.tsx

```diff
@@ -69,6 +71,73 @@ const SortOrderButtonsContainer = styled.div`
 
 function NoteListControls(props: Props) {
 	const searchBarRef = useRef(null);
+	const newNoteRef = useRef(null);
+	const newTodoRef = useRef(null);
+	const noteControlsRef = useRef(null);
+	const searchAndSortRef = useRef(null);
+
+	const [noteBtnText, setNoteBtnText] = useState('New note');
+	const [todoBtnText, setTodoBtnText] = useState('New to-do');
+	const [breakpoint, setBreakpoint] = useState('l');
+	const [iconNote, setIconNote] = useState('fas fa-plus');
+	const [iconTodo, setIconTodo] = useState('fas fa-plus');
```

Comment:
Maybe noteIcon/todoIcon? (for consistency with noteButtonText/todoButtonText)

Reply:
Changed.